src/dsp_tools/utils/shared.py

```diff
@@ -99,7 +103,7 @@ def try_network_action(
             if try_again_later or in_500_range:
                 msg = f"Try reconnecting to DSP server, next attempt in {2 ** i} seconds..."
                 print(f"{datetime.now().isoformat()}: {msg}")
-                logger.error(msg, exc_info=True)
+                logger.error(f"{msg} {action_as_str}", exc_info=True)
```

Comment:
you may want to add `i` here too, so it's more immediately clear on which attempt one is when this log happens